app/code/Magento/Downloadable/view/frontend/templates/catalog/product/links.phtml

```diff
@@ -4,20 +4,25 @@
  * See COPYING.txt for license details.
  */
 ?>
-<?php /* @var $block \Magento\Downloadable\Block\Catalog\Product\Links */ ?>
+<?php
```

Comment:
Please remove not needed closing and opening PHP tags

Reply:
fixed